compiler-rt/lib/scudo/standalone/tsd_exclusive.h

```diff
@@ -27,6 +27,31 @@ struct ThreadState {
 template <class Allocator> void teardownThread(void *Ptr);
 
 template <class Allocator> struct TSDRegistryExT {
+  using ThisT = TSDRegistryExT<Allocator>;
+
+  struct ScopedTSD {
+    ScopedTSD(ThisT &TSDRegistry) {
+      CurrentTSD = TSDRegistry.getTSDAndLock(&IsFallback);
+      DCHECK_NE(CurrentTSD, nullptr);
+    }
+
+    ~ScopedTSD() {
+      if (UNLIKELY(IsFallback))
```

Comment:
The getTSDAndLock function uses UnlockRequired as the name of the parameter, but here you use IsFallback. It's a bit confusing to use a different name. Even though you are using a fallback when not locking, I think it's better to keep using UnlockRequired here.

Reply:
You're right, that does look better. Done.